package.json

```diff
@@ -27,7 +27,8 @@
   },
   "dependencies": {
     "stylelint-config-recommended-scss": "^16.0.1",
-    "stylelint-config-standard": "^39.0.0"
+    "stylelint-config-standard": "^39.0.0",
+    "stylelint-scss": "^6.13.0"
```

Comment:
Let's revert this change as `stylelint-scss` is a dependency of the recommended config, which this config extends.